### PR TITLE
Add Kerberos callback registrar function 

### PR DESCRIPTION
**Pull request by chao-sun-kaazing**

### CODE REVIEW

File: server/src/main/java/org/kaazing/gateway/server/context/resolve/GatewayContextResolver.java [hunk 1]
```diff
@@ -933,6 +933,10 @@ private RealmsContext resolveRealms(SecurityType securityConfig, SecurityContext
                             authType.getHttpCookieArray(),
                             resolveAuthorizationMode(authType.getAuthorizationMode()),
                             authType.getSessionTimeout());
```
Comment (dpwspoon) [inline server/src/main/java/org/kaazing/gateway/server/context/resolve/GatewayContextResolver.java]: Lets also throw a deprecated for the kerberos5.proxy service

File: server/src/main/resources/META-INF/services/org/kaazing/gateway/server/loginModule/gss [hunk 2]
```diff
@@ -1 +1 @@
-class=org.kaazing.gateway.security.auth.gss.GssLoginModule
```
Comment (dpwspoon) [inline server/src/main/resources/META-INF/services/org/kaazing/gateway/server/loginModule/gss]: Shouldn't this be removed from community and put in enterprise?

**Comment by dpwspoon** [comment]
Gave feedback, otherwise LGTM.  @cmebarrow @sanjay-saxena or @jitsni, can one of you also review

File: server/src/main/resources/META-INF/services/org/kaazing/gateway/server/loginModule/gss (hunk 2 above)
Comment (chao-sun-kaazing) [inline server/src/main/resources/META-INF/services/org/kaazing/gateway/server/loginModule/gss]: yes. I moved this file to enterprise.

**Pull request merged**